test/e2e/e2e_test.go

```diff
@@ -69,8 +71,8 @@ var _ = Describe("controller", Ordered, func() {
 
 	Context("Operator", func() {
 		It("should run successfully", func() {
-			config, err := utils.GetConfig()
```

Comment:
delete `utils.GetConfig()` if it is not used any more?

Reply:
Sure, done